Source/DataTypes/SvgOrient.cs

```diff
@@ -82,7 +100,7 @@ public override string ToString()
             string type = string.Empty;
 
 			if (this.IsAuto)
-				return "auto";
+				return this.IsAutoStartReverse ? "auto-start-reverse" : "auto";
```

Comment:
This is actually not in the SVG 1.1 spec, which is used in the lib until now (and the support is not complete by far). I think it is a good idea to add later additions (e.g. SVG 2) to the library, though it would be nice to have an example image for a regression test. Our current W3C test images are for SVG 1.1.
@tebjan - what do you think about SVG 2 support? Maybe we shall a SVG 2 test suite?

Reply:
I did not find any information about the fact that the lib support only the SVG 1.1 spec. Maybe it can be added in the README ?

Reply:
Good point. I guess that at the time the library was created by Microsoft, this has been the latest spec, but a mention in the README would certainly help.